src/firefly/js/templates/lightcurve/LcPhaseFoldingPanel.jsx

```diff
@@ -361,18 +362,29 @@ function resultsSuccess(request) {
 }
 
 function onSearchSubmit(request) {
-    console.log(request);
-    if (request.Tabs==='LC Param') {
-        doPhaseFolding(request);
-    }
-    else {
-        console.log('request no supported');
-    }
+    let fieldState = FieldGroupUtils.getGroupFields('LC_FORM_Panel');
+
+    // TODO Fix: request doesn't contain 'Tabs'...
+    //if (request.Tabs==='LC Param') {
+        doPhaseFolding(fieldState);
+    //}
+    //else {
+    //    console.log('request no supported');
+    //}
 }
 
 //here to plug in the phase folding processor
-function doPhaseFolding(request) {
-    var tReq;
+function doPhaseFolding(fields) {
+    let tbl = getTblById(RAW_TABLE);
+
+    console.log(fields);
+
+    var tReq = makeTblRequest('PhaseFoldedProcessor', PHASE_FOLDED, {
+        'period_days': fields.period.value,
+        'table_name': 'folded_table',
+        'time_col_name':fields.timeCol.value,
+        'original_table': tbl.tableMeta.tblFilePath
```

Comment:
This is because of the workflow we decided to implement somehow. When the LC viewer is displayed, first thing to do is to bring a 'RWA_TABLE' by uploading one. Then it shouldn't complain.